tumblr-photo-video-ripper.py

```diff
@@ -181,9 +181,12 @@ def download_photos(self, site):
 
     def _download_media(self, site, medium_type, start):
         current_folder = os.getcwd()
-        target_folder = os.path.join(current_folder, site)
-        if not os.path.isdir(target_folder):
-            os.mkdir(target_folder)
+        target_folder_base = os.path.join(current_folder, site)
+        if not os.path.isdir(target_folder_base):
+            os.mkdir(target_folder_base)
+        target_folder_other = os.path.join(current_folder, site+'-other')
```

Comment:
Why not using the origin site name as the new folder name? And we could avoid downloading the duplicate photos/videos as well.

I mean if the media in A is reblogged from B, you can just create a new folder B to store this media, instead of current `A-other`.

Reply:
It is not comfortable when threre ara many reposts from many different blogs and a user need to see lot of photos / videos.

Reply:
> It is not comfortable when threre ara many reposts from many different blogs and a user need to see lot of photos / videos...

Still use A and B as an example.
```
A: media1
   media2 (rebloged from B)
B: media2 (rebloged from C)
   media3
```

What if a user is trying to downloading media2 from site A, you will save it to `A-other` folder. And later if downloading from site B, `media2` will be downloaded again and save to `B-other` folder.

Actually they are identical, which should not be save multiple times.

Then why not directly saving it to folder `C` (for site C)?

Reply:
Maybe... I made this feature for me and it's bad for me what you say.